### ISSUE TITLE
Cannot remove Core area in DXA 1.6 web application

### ISSUE THREAD

**Issue by jhorsman** [sdl/dxa-web-application-dotnet#35]
In a DXA 1.6 .NET based webapp [I have created](http://tridion.stackexchange.com/questions/16101/custom-dxa-main-region) my own module, let's call it MyModule. All views, including page views, region views and the main view are in the MyModule area.

I am not dependent on the views and models from the Core area, however, the website does not work without a Core area registration. [This is](https://gist.github.com/jhorsman/7f27899698dd096b88d413750c9b8684) the minimal registration I need to be able to run the site: an `AreaName` and an empty `RegisterAllViewModels()`.

```
namespace MyWebApp.Web.Site.Areas.Core
{
    using Sdl.Web.Common.Models;
    using Sdl.Web.Mvc.Configuration;

    public class CoreAreaRegistration : BaseAreaRegistration 
    {
        public override string AreaName => "Core";

        protected override void RegisterAllViewModels()
        {
        }
    }
}
```

This is a defect, since DXA 1.6 the Core module is split up in Framework and Core module with the purpose of making the core module optional.

I did not investigate in much detail, but I think this issue is related to the `ControllerAreaName`. The controllers are in the DXA Framework, and we still use the ControllerAreaName `Core`. Technically the page Controller is not in any area, but in the DXA framework.

**Comment by rpannekoek** [comment]
TSI-2201 (internal issue ID for tracking purposes)

**Comment by rpannekoek** [comment]
Fixed in DXA 1.7: it is now possible to configure a "default module".
